zh/backup-to-gcs.md

```diff
@@ -134,27 +132,27 @@ GCS 支持以下几种 bucket ACL 策略：
 更多 `Backup` CR 字段的详细解释：
 
 * `.spec.metadata.namespace`：`Backup` CR 所在的 namespace。
+* `.spec.cleanData`：设置为 true 时删除该 Backup CR 时会同时清除该 CR 备份出的数据，默认为 false。值得注意的是，在 v1.1.2 以及之前版本不存在该字段且默认在删除 CR 时同时删除备份的文件。若 v1.1.3 版及之后用户希望保持该行为，需要设置该字段为 true。
 * `.spec.from.host`：待备份 TiDB 集群的访问地址，为需要导出的 TiDB 的 service name，例如 `basic-tidb`。
 * `.spec.from.port`：待备份 TiDB 集群的访问端口。
 * `.spec.from.user`：待备份 TiDB 集群的访问用户。
 * `.spec.from.tidbSecretName`：待备份 TiDB 集群所需凭证的 secret。
 * `.spec.gcs.bucket`：存储数据的 bucket 名字。
 * `.spec.gcs.prefix`：这个字段可以省略，如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.gcs.bucket}/${.spec.gcs.prefix}/backupName`。
-* `.spec.mydumper`：Mydumper 相关的配置，主要有两个字段：一个是 [`options`](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/) 字段，里面可以指定 mydumper 需要的一些参数；一个是 `tableRegex` 字段，可以指定让 Mydumper 备份符合这个正则表达式的表。默认情况下 Mydumper 这个字段可以不用配置。当不指定 Mydumper 的配置时，`options` 和 `tableRegex` 字段的默认值如下：
+* `.spec.dumpling`：Dumpling 相关的配置，主要有两个字段：一个是 [`options`](https://pingcap.com/docs-cn/stable/dumpling-overview/) 字段，里面可以指定 Dumpling 需要的一些参数；一个是 `tableFilter` 字段，可以指定让 Dumpling 备份符合 [table-filter 规则](https://pingcap.com/docs-cn/stable/table-filter/) 的表。默认情况下 dumpling 这个字段可以不用配置。当不指定 dumpling 的配置时，`options` 和 `tableFilter` 字段的默认值如下：
```

Comment:
btw, I can't find the `options` field in the linked page. Is this link valid?

Reply:
perhaps it's better to link (https://pingcap.com/docs-cn/stable/dumpling-overview/) to Dumpling 🤔

Reply:
1. This link (https://pingcap.com/docs-cn/stable/dumpling-overview/) is hosted on the **old** website, which **will be replaced** by the **new** site (https://docs.pingcap.com/zh/tidb/dev/dumpling-overview) very soon. The contents are completely the same on the two pages, so I suggest **using the new one**.
2. The provided link does not have contents related to the `options` field you mentioned here, it might confuse our users if they click on the link and can't see what they expect. Please find the right doc.😃

Reply:
Dumpling has no `options` field 😨 `options` field should be filled with dumpling's arguments, like the following example shows. How to describe this better? 🤔